### ISSUE TITLE
Error when trying to install package from repo via pip

### ISSUE THREAD

**Issue by 0xf0f** [pyminimp3/pyminimp3#1]
Seems there's some issue with cloning the minimp3 submodule
```
C:\WINDOWS\system32>pip install git+https://github.com/pyminimp3/pyminimp3
Collecting git+https://github.com/pyminimp3/pyminimp3
  Cloning https://github.com/pyminimp3/pyminimp3 to c:\users\admin\appdata\local\temp\pip-req-build-e889qr5g
Host key verification failed.
fatal: Could not read from remote repository.

Please make sure you have the correct access rights
and the repository exists.
fatal: clone of 'git@github.com:lieff/minimp3.git' into submodule path 'C:/Users/admin/AppData/Local/Temp/pip-req-build-e889qr5g/minimp3' failed
Failed to clone 'minimp3'. Retry scheduled
Host key verification failed.
fatal: Could not read from remote repository.

Please make sure you have the correct access rights
and the repository exists.
fatal: clone of 'git@github.com:lieff/minimp3.git' into submodule path 'C:/Users/admin/AppData/Local/Temp/pip-req-build-e889qr5g/minimp3' failed
Failed to clone 'minimp3' a second time, aborting
Command "git submodule update --init --recursive -q" failed with error code 1 in C:\Users\admin\AppData\Local\Temp\pip-req-build-e889qr5g
```

**Comment by 0xf0f** [comment]
Reported on the pip repo as well at pypa/pip#6623

**Comment by pganssle** [comment]
@0xf0f For a short term mitigation, you should be able to clone the repo locally and do `git submodule update --all` or whatever the submodule commands are, then `pip install` from there.

I will warn you, though, that one of the reasons I haven't posted this on PyPI yet is that it's still in *very* early stages, and doesn't really have a proper public interface yet, only the low-level bindings are implemented so far.

In any case, thank you for your interest in this module, and hopefully I can get you a more *usable* package soon :wink:

**Comment by 0xf0f** [comment]
Thanks for your response, I'll try setting it up manually that way.

I don't mind it being in early stages. I was mostly just curious and wanted to take a look at your work.
Thank you for making it. I look forward to future releases. :)

**Comment by pganssle** [comment]
Fixed by #5
